pkg/webui/console/store/selectors/application.js

```diff
@@ -39,6 +39,13 @@ import { createErrorSelector } from './error'
 
 const ENTITY = 'applications'
 const ENTITY_SINGLE = 'application'
+const storeSelector = state => state.application
+
+export const applicationSelector = state => storeSelector(state).application || {}
```

Comment:
there is no reason to provide default value, if there is no application in the store we want to return something that evaluates to `false`